evap/evaluation/templates/contact_modal.html

```diff
@@ -48,6 +48,8 @@ <h5 class="modal-title" id="{{ modal_id }}Label">{{ title }}</h5>
     function {{ modal_id }}Show() {
         $('#{{ modal_id }}ActionButton').unbind().click(function(){ {{ modal_id }}Action(); });
         $('#{{ modal_id }}').modal();
+        // remove wrongly added padding by Bootstrap (https://github.com/twbs/bootstrap/blob/v4-dev/js/src/modal.js#L429)
+        $('body, .navbar, .footer').css('padding-right', '');
```

Comment:
you can remove the `.footer` now, it's not fixed anymore.

Reply:
Changed to a more redundant, but future-proof solution  (same selectors like bootstrap)